### ISSUE TITLE
Unable to add devices to HomeKit even they appear in HomebridgeUI

### ISSUE THREAD

**Issue by janmey-meja** [grzegorz914/homebridge-melcloud-control#25]
I am unable to add my two airconditions to HomeKit even they appear properly in Homebridge. I followed the description but the process is timing out in HomeKit with the message “Devise doesn’t respond”. I was running the plug-in from zephyus/ Homebridge-MElcloud2 and there is no need for this manual step in HomeKit to add the devices. As soon the Homebridge is connected to HomeKit my two units appear in HomeKit. Maybe you can contact him how he pulled this off in his plugin. 

https://github.com/zephyrus/homebridge-melcloud2

Thanks for your effort

**Comment by grzegorz914** [comment]
Hi,

This plugin is a dynamic external accessory and use other way to add it to HomeKit.
Are You realised this steps:
https://github.com/grzegorz914/homebridge-melcloud-control/wiki#adding-to-homekit https://github.com/grzegorz914/homebridge-melcloud-control/wiki#troubleshooting

Note!!!
You can only use one plugin at same time, firs remove other MELCloud plugin and its cache from homebridge before add this plugin.

**Comment by janmey-meja** [comment]
Yes I was following these steps, removed the other plugin from Homebridge, deleted the bridge ad it’s devices from HomeKit. Restarted Homebridge and installed the new plugin. Added the Bridge again to HomeKit and tried to add my two devices who are shown after choosing manually but the last step fail’s. 
![6398AD99-484A-4F5F-ACEA-8AA8C4A2F531](https://user-images.githubusercontent.com/104717062/209655856-4bf61a24-e185-4934-b249-eaa7a569d6f6.jpeg)
![5DC4ED13-4BB3-4046-9A0C-B7DDA1307A02](https://user-images.githubusercontent.com/104717062/209655865-1cf0b52c-8d51-4f29-b571-1bc449f581de.jpeg)
![02A45FA9-A4C4-42C4-BCE6-81D9797BBD3C](https://user-images.githubusercontent.com/104717062/209655867-9380e61a-296c-4f83-bfcd-27587bc1cec4.jpeg)
Tried it yesterday and again today but no success. Added now the older plugin again and they MEL devices appear right in HomeKit without any additional action needed.

**Comment by grzegorz914** [comment]
Can You enable debug mode in plugin config advanced section and get the complete log durig this proces.

**Comment by janmey-meja** [comment]
I send the log files per direct email as they contain personal information.

**Comment by grzegorz914** [comment]
The log and all data is correct, I think the response time from device is too long and pairing fail.
I have 5 A.C devices and all paired correct without any problem, also other users do not report any Probleme with pairing.
What version of Homebridge You use? May be try pairing with other device, reboot iDevice, reboot network, etc..

**Comment by janmey-meja** [comment]
I use HomeBridge v1.6.0 with node.js v16.16.0 on a Synology NAS in Switzerland. The connection from MELCloud to the actual AirCon is via poor German mobile network. I switched now back to the plugin from Kristof and it paired immediately. 
I will send you the complete log per email as I have at least learned now how to get it from Homebridge. I am an old men but still learning. I have tried pairing on iPhone, iPad and Mac. Internet speed from my NAS to MELCloud should not be an issue, the only slow connection is the 3rd world German mobile connection from the Aircon to MELCloud.
